Convert React-Web components to web-components

zarkoboyadjiev-zz opened the pull request:
Convert non-children related React-web components ( except Icon component ) to web-components & add storybook examples.
Edit component seems to have some problem and doesn't work.
Closes #67

zarkoboyadjiev committed:
feat(web-components): convert Image from react-web & add Storybook example

zarkoboyadjiev committed:
refactor(web-components): convert Input from react-web & add Storybook example

zarkoboyadjiev committed:
refactor(web-components): convert Menu from react-web & add Storybook example

zarkoboyadjiev committed:
refactor(web-components): convert Modal from react-web & add Storybook example

zarkoboyadjiev committed:
refactor(web-components): convert ProgressBar from react-web & add Storybook example

zarkoboyadjiev committed:
refactor(web-components): convert Select from react-web & add Storybook example

zarkoboyadjiev committed:
refactor(web-components): convert Uploader from react-web & add Storybook example

zarkoboyadjiev committed:
refactor(web-components): convert Thumbnail

react-web component doesn't work and need to be fixed

zarkoboyadjiev committed:
refactor(web-components): convert Logo from react-web

zarkoboyadjiev committed:
refactor(web-components): update main index file

zarkoboyadjiev committed:
refactor(General): fix code editor and add new line where was needed

kidroca commented on packages/components/web-components/src/Modal/Modal.stories.js:
```diff
@@ -0,0 +1,69 @@
+import { action } from '@storybook/addon-actions';
+
+import './Modal';
+import image from './assets/photo.jpg';
```
Maybe we should use an image from the web instead of copying this 12MB photo

kidroca reviewed (commented):
I've saw some external links used in the image component stories, maybe we can do the same for the Photo modal, so that we don't copy this big file across packages

kidroca committed:
fix(react-web): Input/Edit Fragment usage fix

Using the fragment shorthand <> does not translate to `web-components`

Related to #76

kidroca reviewed (approved):
Looks good, opened some specific issues we need to address after this is merged
